draft-ietf-oauth-jwt-introspection-response.xml

```diff
@@ -214,155 +214,154 @@ token=2YotnFZFEjr1zCsicMWpAA]]></artwork>
+         server.</t>
+         <t hangText="aud">MUST identify the resource server receiving the token
+         introspection response.</t>
+         <t hangText="token_introspection">A JSON object containing the members
+         of the token introspection response, as specified in the "OAuth Token
+         Introspection Response" registry established by <xref target="RFC7662"/>
+         as well as other members. The separation of the introspection members into
+         a dedicated containing JWT claim is intended to prevent conflict and confusion
+         with top-level JWT claims that may bear the same name.
+         <vspace blankLines="1" />
+         If the access token is invalid, expired, revoked, or is not intended for the
+         calling resource server (audience), the authorization server MUST set the value of the
+         <spanx style="verb">active</spanx> member in the <spanx style="verb">token_introspection</spanx>
+         claim to <spanx style="verb">false</spanx>. Otherwise, the <spanx style="verb">active</spanx>
+         member is set to <spanx style="verb">true</spanx>.
+         <vspace blankLines="1" />
+         If possible, the AS MUST narrow down the <spanx style="verb">scope</spanx> value
+         to the scopes relevant to the particular RS.
+         <vspace blankLines="1" />
+         Claims from the "JSON Web Token Claims" registry that are commonly used in
+         <xref target="OpenID.Core"/> and can be applied to the resource owner MAY be included
+         as members in the <spanx style="verb">token_introspection</spanx> claim.
+         They can serve to identify the resource owner as a natural person or to
+         provide a required contact detail, such as an e-Mail address or phone number.
+         When transmitting such claims the AS acts as an identity provider in regard to
+         the RS. The AS determines based on its RS-specific policy what claims about the
+         resource owner to return in the token introspection response.
+         <vspace blankLines="1" />
+         The AS MUST ensure the release of any privacy-sensitive data is legally based.
+         <vspace blankLines="1" />
+         Further content of the introspection response is determined by the RS-specific
+         policy at the AS.</t>
+     </list>
+     </t>
+
+     <t>The JWT MAY include other claims, including those from the "JSON Web Token Claims"
+     registry established by <xref target="RFC7519"/>, such as <spanx style="verb">iat</spanx>
+     to indicate when the introspection response was issued by the AS. The JWT SHOULD NOT
+     include the <spanx style="verb">sub</spanx> and <spanx style="verb">exp</spanx> claims
+     as an additional prevention against misuse of the JWT as an access token (see
+     <xref target="Cross-JWT_Confusion"/>).</t>
           
      <t>Note: Although the JWT format is widely used as an access token format, the JWT
```

Comment:
do we still need this comment?

Reply:
Do you mean the note "Although the JWT format is widely used as an access token format, the JWT returned in the introspection response is not an alternative representation of the introspected access token and is not intended to be used as an access token"?

I suggest it stays because regardless of what the cross JWT confusion paragraph says, there will be people still thinking of that :)